Core/Core.csproj

```diff
@@ -10,9 +10,12 @@
     <PackageReference Include="Castle.Core" Version="4.4.0" />
     <PackageReference Include="FluentValidation" Version="11.2.2" />
     <PackageReference Include="log4net" Version="2.0.15" />
+    <PackageReference Include="MediatR" Version="11.0.0" />
```

Comment:
Why do you need `MediatR` dependency here?

Reply:
First, I tried to do with CQRS but then I gave up. I'll remove it :)